tests/support/identity.rs

```diff
@@ -0,0 +1,164 @@
+use support::*;
+
+use std::{
+    collections::VecDeque,
+    fs, io,
+    path::{Path, PathBuf},
+    sync::{Arc, Mutex},
+    time::{Duration, SystemTime},
+};
+
+use linkerd2_proxy_api::identity as pb;
+
+pub struct Identity {
+    pub env: app::config::TestEnv,
+    pub certify_rsp: pb::CertifyResponse,
+}
+
+#[derive(Clone)]
+pub struct Controller {
+    expect_calls: Arc<Mutex<VecDeque<Certify>>>,
+}
+
+type Certify = Box<
+    Fn(
+            pb::CertifyRequest,
+        )
+            -> Box<Future<Item = grpc::Response<pb::CertifyResponse>, Error = grpc::Status> + Send>
+        + Send,
+>;
```

Comment:
I want to note here that I did not choose this formatting, this is what `rustfmt` thought would look good.

Reply:
![wat](https://i.imgur.com/IppKJ.jpg)